core/src/Kokkos_NumericTraits.hpp

```diff
@@ -364,16 +364,9 @@ struct reduction_identity<signed char> {
   }
 };
 
+// 4707 fix; effectively reverting much of PR #4352 
```

Comment:
I don't think it adds any value and would just remove.
```suggestion
```

Reply:
Oh, right.  That was just a "note to self" that I forgot to remove.